spring-cloud-gateway-webflux/src/main/java/org/springframework/cloud/gateway/webflux/ProxyExchange.java

```diff
@@ -132,6 +132,8 @@
 
 	private Set<String> sensitive;
 
+	private Set<String> insensitive;
```

Comment:
This is not part of spring-cloud-gateway-server and should be removed from this enhancement since this module doesn't know anything about server properties.